synapse/daemon.py

```diff
@@ -117,18 +136,22 @@ class Daemon(s_base.Base):
         self.listenservers = [] # the sockets we're listening on
         self.connectedlinks = [] # the links we're currently connected on
 
+        self.sessions = {}
+
         self.mesgfuncs = {
             'tele:syn': self._onTeleSyn,
             'task:init': self._onTaskInit,
             'share:fini': self._onShareFini,
+
+            # task version 2 API
+            't2:init': self._onTaskV2Init,
```

Comment:
do we need to rev the minor number of the telepath api here?

Reply:
I guess we could, but as is it is 100% backward compatible *and* forward detectable....